components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2ScopeService.java

```diff
@@ -101,8 +101,11 @@ public Scope registerScope(Scope scope)
                         ERROR_CODE_FAILED_TO_GET_ALL_SCOPES, e);
             }
         } else {
-            //check if it is a pagination request.
-            scopes = listScopesWithPagination(startIndex, count);
+            //avoid null pointer dereference
+            //if (startIndex != null && count != null) {
```

Comment:
why these are commented here?
Without the comments there is no change to the Original.